generate-latest-dockerfile.sh

```diff
@@ -77,7 +77,7 @@ for arch in $archs; do
 		# Debian.
 		# For armv7hf and armv6hf, if node version is less than 4.x.x (0.10.x 0.12.x) then that image will use binaries from resin, otherwise it will use binaries from official distribution.
```

Comment:
Can you change this comment to reflect the code? i.e if node version is **greater or equal than 3.x.x**...